### PR TITLE
Feature/page header

### PR DESCRIPTION
**Pull request by MaryanaMysak**
Знімок екрана 2021-06-25 о 16 05 47

### CODE REVIEW

**Review by StepanNaryshkovEleks — CHANGES_REQUESTED** [review]

File: src/components/header/component.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import styles from "./_.module.css";
+
+export const Header = ({title, LeftComponent, RightComponent, backgroundColor}) => {
```
Comment (StepanNaryshkovEleks) [inline src/components/header/component.js:4]: image
image

File: src/components/header/_.module.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+.icon {
+  width: 17px;
+  height: 17px;
+}
+
+.title {
+  font-size: 18px;
+  font-weight: 400;
+  margin-bottom: 3px;
```
Comment (StepanNaryshkovEleks) [inline src/components/header/_.module.css:16]: why do we need margin bottom here?
image
looks like we have different margin, so I think we can delete it

File: src/components/header/_.module.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.icon {
+  width: 17px;
+  height: 17px;
+}
+
+.title {
+  font-size: 18px;
+  font-weight: 400;
```
Comment (StepanNaryshkovEleks) [inline src/components/header/_.module.css:15]: Looks like redundant value

File: src/components/header/_.module.css (hunk 2 above)
Comment (MaryanaMysak) [reply src/components/header/_.module.css:16]: It's for vertical-align of title relatively to image and icon

**Pull request merged**